mergify_engine/web/api/queues.py

```diff
@@ -317,5 +324,435 @@ async def repository_queues(
                 )
 
             queues.queues.append(queue)
-
         return queues
+
+
+QueuePauseSchema = voluptuous.Schema({voluptuous.Required("message"): str})
+
+
+@router.post(
+    "/repos/{owner}/{repository}/queue/{queue_name}/pause_from",  # noqa: FS003
```

Comment:
why `_from`?

Reply:
Because of the "falling" behavior of the pause system that we discussed, I thought initially that "we pause all queues from the one requested to the last queue behind in processing order of the queues". Formalized like this, I thought adding "from" would be more intuitive to understand the behavior, but it's related to @sileht question above about if we need just one API call of multiple call to pause several queues